skeleton-typescript-webpack/src/main.ts

```diff
@@ -0,0 +1,14 @@
+﻿import {Aurelia} from 'aurelia-framework';
+import {bootstrap} from 'aurelia-bootstrapper-webpack';
```

Comment:
_references.ts file is not important, just files that are right now referenced from there need to be passed to the typescript compiler - see my improvements for this pull request: https://github.com/arjendeblok/skeleton-navigation/pull/1